cli/api/interfaces.go

```diff
@@ -119,4 +120,7 @@ type API interface {
 
 	// Volumes
 	GetVolumeStatus() (*volume.Statuses, error)
+
+    // Public endpoints
+    AddPublicEndpointPort(serviceid, endpointName, portAddr string, usetls bool, protocol string, isEnabled bool) (*servicedefinition.Port, error)
```

Comment:
Yeah. this is my editor.  I was noticing it also in other parts of the code.  I'll work on sorting that out.